ansible/roles/ceph-grafana/tasks/grafana_plugins.yml

```diff
@@ -15,3 +15,6 @@
 - name: Update Grafana plugins
   command: "grafana-cli plugins update {{ item }}"
   with_items: "{{ grafana_plugins }}"
+
+- name: Update status-panel for readability within the 'light' theme
+  command: "sed -i.bak -e 's/green/rgb(1,167,1)/g' /var/lib/grafana/plugins/vonage-status-panel/dist/css/status_panel.css"
```

Comment:
This doesn't look quite right: 
<img width="284" alt="screen shot 2017-06-29 at 10 34 14 am" src="https://user-images.githubusercontent.com/58307/27699148-8cc19052-5cb6-11e7-89c1-3c2c2cd721e4.png">